sdk/src/types/block/slot/commitment.rs

```diff
@@ -18,6 +18,9 @@ use crate::{
     serde(rename_all = "camelCase")
 )]
 pub struct SlotCommitment {
```

Comment:
A new field has been added since, RMC

Reply:
ffs

Reply:
You don't have to do it here, just mentioning since you already added one